### PR TITLE
Added MongoDB support

### LINKED ISSUE #8: Switch to MongoDB
MongoDB seems much better fit for the purposes of CSGOItemDB, and it would certainly look a lot cleaner.

### PR DESCRIPTION
**Pull request by mvegter**
This should fix #8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,8 +4,9 @@
   "description": "A free API to get the most accurate CSGO prices for high tier and low tier items.",
   "main": "server.js",
   "dependencies": {
-    "express": "4.13.3",
     "body-parser": "1.13.3",
+    "mongodb": "^2.2.24",
+    "mongoose": "^4.8.5",
     "mysql": "2.9.0",
```
Comment (andrewda) [inline package.json]: Remove unused dependency.

**Review by andrewda — CHANGES_REQUESTED** [review]
Thanks for this, looks good! I'll test it out in a bit.

**Comment by mvegter** [comment]
@andrewda Should be fixed.

File: models/Keys.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+const mongoose = require('mongoose');
+
+module.exports = mongoose.model('Keys', {
+	key: String,
+	premium: String
```
Comment (andrewda) [inline models/Keys.js]: Make `premium` a boolean

**Review by andrewda — CHANGES_REQUESTED** [review]

File: models/Keys.js (hunk 2 above)
Comment (mvegter) [reply models/Keys.js]: Changed it

**Comment by mvegter** [comment]
This can also fix #13, as it is working as far as I was able to test.

**Review by andrewda — APPROVED** [review]

**Pull request merged**